src/less/tab/base/tab.less

```diff
@@ -7,11 +7,17 @@ div.tabs__items[role="tablist"],
 ul.fake-tabs__items {
     display: flex;
     flex-direction: row;
+    font-size: 14px;
```

Comment:
Not sure, but are there LESS vars that can be used here or not?

Reply:
Hard to spot in the diff, but:

```
// font-size should not scale with REM
```

Reply:
oh git it 👍